src/DotNetDelice/Output.fs

```diff
@@ -40,12 +47,12 @@ let private prettyPrinter printable =
     colorprintfn "│ ├── Is FSF Free/Libre: $green[%b]" printable.IsFsf
     colorprintfn "│ └── Included deprecated IDs: $green[%b]" printable.IsDeprecatedType
     printfn "└─┬ Packages:"
-    printable.Names
+    printable.Packages
     |> Seq.iteri (fun i l ->
            let prefix =
                if i = (printable.Count - 1) then "└"
                else "├"
-           printfn "  %s── %s" prefix l)
+           printfn "  %s── %s" prefix l.Name)
```

Comment:
The pretty print output has now lost the URL from the output as a result of the change. I would prefer it kept that in.

Reply:
pretty print should now be fixed